### PR TITLE
Adds script to output models in yaml or json format.

### PR DESCRIPTION
**Pull request by gregcube**

### CODE REVIEW

File: scripts/generate-models.php [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  else if ($format === 'json') {
+    $output = $serializer->toJson($model);
+    $ext = '.json';
+  }
+
+  $bytes = file_put_contents($path . $model->label() . $ext, $output);
+
+  if ($bytes === false) {
+    throw new RuntimeException('Failed to write model file.');
+  }
```
Comment (lehors) [inline scripts/generate-models.php:51]:
```suggestion
  } else {
    print "Generated models/" . $model->label() . $ext . PHP_EOL;
  }
```

**Review by lehors — COMMENTED** [review]
Thanks. This works fine but I suggest adding some output because the lack of any output when I ran it made me wonder if had done anything at all. :-)

File: scripts/generate-models.php (hunk 1 above)
Comment (gregcube) [reply scripts/generate-models.php:51]: Added.  Just slightly different than your suggestion.

Comment (lehors) [reply scripts/generate-models.php:51]: That's fine. Thanks!

**Review by lehors — APPROVED** [review]
LGTM

**Pull request merged**